grammars/css.cson

```diff
@@ -20,294 +20,1237 @@
+              '0':
+                'name': 'punctuation.definition.string.end.css'
+            'name': 'string.quoted.double.css'
+            'patterns': [
+              {
+                'begin': '(?:\\G|^)(?=(?:[^"])+$)'
+                'end': '$'
+                'name': 'invalid.illegal.unclosed.string.css'
+              }
+            ]
+          }
+        ]
+      }
+      {
+        # @import
+        'begin': '(?i)((@)import)(?:\\s+|$|(?=[\'"]|/\\*))'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.import.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': ';|(?=})'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.terminator.rule.css'
+        'name': 'meta.at-rule.import.css'
+        'patterns': [
+          {
+            'begin': '\\G\\s*(?=/\\*)'
+            'end': '(?<=\\*/)\\s*'
+            'patterns': [
+              {
+                'include': '#comment-block'
+              }
+            ]
+          }
+          {
+            'include': '#string'
+          }
+          {
+            'include': '#url'
+          }
+          {
+            'include': '#media-query-list'
+          }
+        ]
+      }
+      {
+        # @font-face
+        'begin': '(?i)((@)font-face)(?=\\s*|{|/\\*|$)'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.font-face.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': '(?!\\G)'
+        'name': 'meta.at-rule.font-face.css'
+        'patterns': [
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#escapes'
+          }
+          {
+            'include': '#rule-list'
+          }
+        ]
+      }
+      {
+        # @media
+        'begin': '(?i)(?=@media(\\s|\\(|/\\*|$))'
+        'end': '}|(?=;)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.property-list.end.bracket.curly.css'
+        'patterns': [
+          {
+            'begin': '(?i)(@)media(?=\\s|\\(|/\\*|$)'
+            'beginCaptures':
+              '0':
+                'name': 'keyword.control.at-rule.media.css'
+              '1':
+                'name': 'punctuation.definition.keyword.css'
+            'end': '(?=\\s*[{;])'
+            'name': 'meta.at-rule.media.css'
+            'patterns': [
+              {
+                'include': '#media-query-list'
+              }
+            ]
+          }
+          {
+            'begin': '{'
+            'beginCaptures':
+              '0':
+                'name': 'punctuation.section.property-list.begin.bracket.curly.css'
+            'end': '(?=})'
+            'patterns': [
+              {
+                'include': '$self'
+              }
+            ]
+          }
+        ]
+      }
+      {
+        # @page
+        'begin': '(?i)(@)page(?=[\\s:{;]|/\\*|$)'
```

Comment:
No excuses for this, I was probably just being cautious with regard to accidental highlighting spillage. Which, come to think of it, is probably the right way to alert an author to an incorrect block. :p Will fix.